docs/running-headscale-container.md

```diff
@@ -6,72 +6,98 @@
 
 ```shell
 touch ./config/db.sqlite
 ```
 
-3. **(Strongly Recommended)** Download a copy of the [example configuration][config-example.yaml](https://github.com/juanfont/headscale/blob/main/config-example.yaml) from the headscale repository.
-
-Using wget:
+# 3. Create an ACL file in the headscale directory:
 
 ```shell
-wget -O ./config/config.yaml https://raw.githubusercontent.com/juanfont/headscale/main/config-example.yaml
+touch ./config/ACL.json
 ```
 
-Using curl:
+Edit the file and put a base ACL in it :
 
-```shell
-curl https://raw.githubusercontent.com/juanfont/headscale/main/config-example.yaml -o ./config/config.yaml
+```json
+{
+    "acls": [
+      { "action": "accept", "src": ["*:*"], "dst": ["*:*"] }
+    ]
+}  
 ```
 
-**(Advanced)** If you would like to hand craft a config file **instead** of downloading the example config file, create a blank `headscale` configuration in the headscale directory to edit:
+# 4. Download a copy of the [example configuration][config-example.yaml](https://github.com/juanfont/headscale/blob/main/config-example.yaml) from the headscale repository and place it at ./config/config.yaml
 
 ```shell
-touch ./config/config.yaml
+wget -O ./config/config.yaml https://raw.githubusercontent.com/juanfont/headscale/main/config-example.yaml || curl https://raw.githubusercontent.com/juanfont/headscale/main/config-example.yaml -o ./config/config.yaml
 ```
 
 Modify the config file to your preferences before launching Docker container.
-Here are some settings that you likely want:
+Here are some settings that you likely want to change:
+
+Change `Headscale's` server url to your hostname or host IP
 
 ```yaml
-# Change to your hostname or host IP
 server_url: http://your-host-name:8080
-# Listen to 0.0.0.0 so it's accessible outside the container
-metrics_listen_addr: 0.0.0.0:9090
-# The default /var/lib/headscale path is not writable in the container
-private_key_path: /etc/headscale/private.key
-# The default /var/lib/headscale path is not writable in the container
-noise:
-  private_key_path: /etc/headscale/noise_private.key
-# The default /var/lib/headscale path is not writable  in the container
-db_type: sqlite3
-db_path: /etc/headscale/db.sqlite
 ```
 
-4. Start the headscale server while working in the host headscale directory:
+Listen to 0.0.0.0 so the services are accessible outside the container
+
+```yaml
+listen_addr: 0.0.0.0:8080 #For Headscale
+metrics_listen_addr: 0.0.0.0:9090 #For the metrics 
+```
+
+Bind the ACL file we created above:
+
+```yaml
+acl_policy_path: "/etc/headscale/ACL.json"
+```
+
+!!! warning "/var/lib isn't an editable path in the container"
+
+    Because /var/lib isn't an editable path in the container, we will need to replace
+    in the config file all those /var/lib paths to /etc.
+    
+    Example :
+    ```yaml
+    # The default /var/lib path is not writable in the container
+    private_key_path: /etc/headscale/private.key
```

Comment:
Same thing needs to be done for the noise key.

Reply:
I'm not familiar with that concept, I'm gonna look at it.

Reply:
The noise key should also be in a writeable directory. 
```yaml
noise:
  private_key_path: /etc/headscale/noise_private.key
```

Reply:
I'm not sure, but is the noise key optional ? I can't find any information in the documentation

Reply:
The Noise key is absolutely required :)